scripts/imgtool/image.py

```diff
@@ -477,8 +477,9 @@ def create(self, key, public_key_format, enckey, dependencies=None,
                             backend=default_backend())
             encryptor = cipher.encryptor()
             img = bytes(self.payload[self.header_size:])
-            self.payload[self.header_size:] = \
-                encryptor.update(img) + encryptor.finalize()
+            if not clear:
+                self.payload[self.header_size:] = \
+                    encryptor.update(img) + encryptor.finalize()
```

Comment:
If you're not going to use the `encryptor` when `clear` is set, is there a reason to leave the above cryptographic primitive instantiation enabled?

Reply:
No reason,  I ll fix that soon